dev-cpp/gtest/gtest-1.10.0_p20200702.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Python is required for tests and some build tasks.
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib python-any-r1
+
+GOOGLETEST_COMMIT=aee0f9d9b5b87796ee8a0ab26b7587ec30e8858e
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/google/googletest"
+else
+	if [[ -z ${GOOGLETEST_COMMIT} ]]; then
+		URI_PV=v${MY_PV:-${PV}}
+	else
+		URI_PV=${MY_PV:=${GOOGLETEST_COMMIT}}
+	fi
+	SRC_URI="https://github.com/google/googletest/archive/${URI_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+	S="${WORKDIR}"/googletest-${MY_PV}
+fi
+
+DESCRIPTION="Google C++ Testing Framework"
+HOMEPAGE="https://github.com/google/googletest"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="doc examples test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( ${PYTHON_DEPS} )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.10.0_p20200610-increase-clone-stack-size.patch
```

Comment:
```
 * gtest-1.10.0_p20200702.tar.gz BLAKE2B SHA512 size ;-) ...                                 [ ok ]
 * Using python3.7 to build
>>> Unpacking source...
>>> Unpacking gtest-1.10.0_p20200702.tar.gz to /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/work
>>> Source unpacked in /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/work
>>> Preparing source in /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/work/googletest-aee0f9d9b5b87796ee8a0ab26b7587ec30e8858e ...
 * Applying gtest-1.10.0_p20200610-increase-clone-stack-size.patch ...
/var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/temp/environment: line 1328: /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/files/gtest-1.10.0_p20200610-increase-clone-stack-size.patch: No such file or directory
/var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/temp/environment: line 1331: /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/files/gtest-1.10.0_p20200610-increase-clone-stack-size.patch: No such file or directory                                                                                 [ !! ]
 * ERROR: dev-cpp/gtest-1.10.0_p20200702::testworld failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/dev-cpp/gtest-1.10.0_p20200702/files/gtest-1.10.0_p20200610-increase-clone-stack-size.patch
```

Reply:
Amended locally but forgot to push.  It should be OK now.